inbm/dispatcher-agent/dispatcher/sota/constants.py

```diff
@@ -18,7 +18,7 @@
 TIBER_UPDATE_TOOL_PATH = get_canonical_representation_of_path('/usr/bin/os-update-tool.sh')
 
 # Release server access token path
-RELEASE_SERVER_TOKEN_PATH = get_canonical_representation_of_path('/etc/intel_edge_node/tokens/platform-update-agent/'
+RELEASE_SERVER_TOKEN_PATH = get_canonical_representation_of_path('/etc/intel_edge_node/tokens/release-service/'
```

Comment:
according to Matenko, Tadeusz: "The correct path is /etc/intel_edge_node/tokens/release-service/access_token"

Reply:
Path changed to /etc/intel_edge_node/tokens/release-service/access_token